### PR TITLE
Renamed every file

### PR DESCRIPTION
**Pull request by bryanbell**
files with two dot extensions cause trouble.

### CODE REVIEW

**Comment by pdehaan** [comment]
@nchapman Want to give this a final look before I merge in the AM?
Once this lands, I can rebase my other PRs since this will no doubt cause a big :boom:.

File: top_sites.json [hunk 1]
```diff
@@ -1,584 +1,572 @@
...
+    "image_url": "images/macys-com.png",
+    "background_color": "#000"
   },
   {
-    "title": "Xfinity",
-    "url": "http://www.xfinity.com/",
-    "image_url": "images/comcast.net.png",
-    "background_color": "#777"
+    "title": "Mail Online",
+    "url": "http://www.dailymail.co.uk/ushome/index.html",
```
Comment (pdehaan) [inline top_sites.json:322]: Do we have strong feelings one way or the other on some of these locale specific URLs?
I'm mildly leaning more towards just having the domains and trying to leave off anything US/region specific, but I could easily be convinced to care less.

File: top_sites.json [hunk 2]
```diff
@@ -1,584 +1,572 @@
...
+    "url": "http://patch.com",
+    "image_url": "images/patch-com.png",
+    "background_color": "#499341"
   },
   {
-    "title": "USPS",
-    "url": "https://www.usps.com/",
-    "image_url": "images/usps.com.png",
-    "background_color": "#0F6FAB"
+    "title": "Paypal",
```
Comment (pdehaan) [inline top_sites.json:381]: PayPal

File: top_sites.json [hunk 3]
```diff
@@ -1,584 +1,572 @@
...
   },
   {
-    "title": "Gfycat",
-    "url": "http://gfycat.com/",
-    "image_url": "images/gfycat.com.png",
-    "background_color": "#F6F8FC"
+    "title": "Salesforce.com",
+    "url": "http://www.salesforce.com/platform/products/force/",
+    "image_url": "images/force-com.png",
+    "background_color": "#000"
```
Comment (pdehaan) [inline top_sites.json:414]: Why two Salesforce.com results? I feel like we should maybe keep the first one, and remove this entry, unless we think ... yeah, we should probably just remove this oddly specific entry.

**Comment by nchapman** [comment]
@pdehaan I think we should merge this now and make additional changes in follow up PRs. Thoughts?

File: top_sites.json (hunk 1 above)
Comment (nchapman) [inline top_sites.json:322]: Those URLs are all from embedly based on where it was redirected to when visiting the raw domain with `http://` in front. Let's trim them all down to not be country specific.

Comment (pdehaan) [inline top_sites.json:322]: Here's my quick list of all URLs which have a non-blank `pathname`:

https://www.blogger.com/home => /home
http://www.buzzfeed.com/index => /index
https://online.citi.com/US/Welcome.c => /US/Welcome.c
http://www.wikia.com/fandom => /fandom
http://www.forbes.com/home_usa/ => /home_usa/
http://www.dailymail.co.uk/ushome/index.html => /ushome/index.html
https://www.paypal.com/home => /home
http://www.salesforce.com/platform/products/force/ => /platform/products/force/
https://www.washingtonpost.com/regional/ => /regional/
http://www.webmd.com/default.htm => /default.htm

I'll do quick scan of those and see what paths we can safely remove.

**Pull request merged**